site/content/installation/running-on-kind.md

```diff
@@ -40,13 +40,59 @@ To install NGINX Gateway Fabric, choose the appropriate installation guide that
 - [Installation with Helm]({{< relref "installation/installing-ngf/helm.md" >}})
 - [Installation with Kubernetes manifests]({{< relref "installation/installing-ngf/manifests.md" >}})
 
+## Set up a NodePort
+
+When using kind clusters, be aware that NodePort services require [additional setup](https://kind.sigs.k8s.io/docs/user/configuration/#nodeport-with-port-mappings).
+
+For example, the following will automatically set up port forwarding into a local cluster (intended for development):
+
+```yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraPortMappings:
+  - containerPort: 31437
+    hostPort: 8080
+    protocol: TCP
+  - containerPort: 31438
+    hostPort: 8443
+    protocol: TCP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx-gateway
+  namespace: nginx-gateway # must be same namespace as your gateway
+  labels:
+    app.kubernetes.io/name: nginx-gateway
+    app.kubernetes.io/instance: nginx-gateway
+    app.kubernetes.io/version: "1.3.0"
```

Comment:
```suggestion
    app.kubernetes.io/version: "edge"
```

Reply:
@kate-osborn Where would a person find the correct version if they were not defaulting to `edge` and how could that be expressed as a comment?

Reply:
We essentially have two versions of our documentation:

- `edge`: docs on the `main` branch in github
- `release`: docs on docs.nginx.com

When we release, we create a release branch and change all the references to `edge` to the release version. We then push these docs to docs.nginx.com.

Documentation in the `main` branch should always refer to `edge`.

 For example, the [NodePort Service](https://github.com/nginxinc/nginx-gateway-fabric/blob/main/deploy/manifests/service/nodeport.yaml) manifest on the `main` branch  points to the `edge` version. 

Whereas, the [same manifest on the release-1.3 branch](https://github.com/nginxinc/nginx-gateway-fabric/blob/release-1.3/deploy/manifests/service/nodeport.yaml) points to `1.3.0`.

Reply:
Thanks, I've added a short comment that I think will at least signal the intent to user.

Reply:
Please resolve if acceptable.

Reply:
Labels are just used for informational purposes. Technically, it does not need to match the version of NGF deployed. Personally, I think the comment is unnecessary and slightly confusing. @sjberman thoughts?

Reply:
Agreed, we don't need the comment

Reply:
Labels are often used for selection. Without some indication that these are not going to be their significance is up in the air, especially for beginners. 

Perhaps they could be removed entirely from the example? Or perhaps a comment like:

`# labels following Recommended Labels guidance https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/`? It is a tutorial after all.

Anyway, I'll amend.

Reply:
Have pushed up a change.